vaadin-grid-flow-parent/vaadin-grid-flow/src/test/java/com/vaadin/flow/component/grid/GridSortingTest.java

```diff
@@ -263,6 +263,14 @@ public void template_renderer_non_comparable_property() {
                 grid.getDataCommunicator().getBackEndSorting());
     }
 
+    @Test
+    public void checkMultiSort() {
```

Comment:
I suggest to separate two cases, one correctly reading the initial value, the other reading the correct value after we change it via code.

Reply:
Done.